### ISSUE TITLE
validate_ami_bags.py. / ami_bagError questions

### ISSUE THREAD

**Issue by genfhk** [NYPL/ami-tools#33]
1. This error doesn't get conveyed in the error/warning 'summary' (are all errors supposed to be summarized?): 
```ERROR - Following error encountered while loading /Volumes/NYPL357540/Video/492776: 'AMI bag must contain either Excel or JSON metadata'```
2. In one instance, a bag threw a "Missing metadata files" message which was not also caught by the "...must contain either Excel or JSON metadata" ami_bagError - so although it was missing metadata for some SCs, this bag only threw a 'warning', rather than an error. 
```WARNING - Error in bag structure: Filenames for media files do not match filenames for metadata.``` ...``` Missing metadata files:``` 
3. Warning that should be an error for most video formats (I think optical video would be the only exception? Should we ask vendors to quarantine optical video as a workaround?) : 
```Error in asset balance: Mismatch of PM's and SC's```

General thing: Use of the word "error" in Warning message makes parsing difficult.

**Comment by nkrabben** [comment]
1. Should be handled with last couple PRs, but I will build a test case on Monday.
2. Updated logger to throw an error. This case shouldn't through the first kind of error since that checks for the existence of any metadata. This check is for the proper pairing of one json file for every file and in a folder together. I found a case like this recently where the SC JSONs were in the PM folder. There's now an additional check to report if that's the case. You'll see:
`ERROR - Error in bag structure: Files in the wrong directory: <relative payload path to file>`
3. warning has been updated
